bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -81,31 +96,14 @@
     }}
 
     // Try loading a local version of requirejs and fallback to cdn if not possible.
-    script_load(base_url + 'static/scripts/JSRoot.core.js', script_success, function(){{
-        console.error('Fail to load JSROOT locally, please check your jupyter_notebook_config.py file')
-        script_load('https://root.cern/js/6.1.0/scripts/JSRoot.core.min.js', script_success, function(){{
+    script_load_{jsDivId}(base_url + 'static/scripts/JSRoot.core.js', function(){{
```

Comment:
Why is `script_success` no longer necessary?

Reply:
I just add drawing of the object directly